.travis.yml

```diff
@@ -27,7 +27,6 @@ jobs:
         - npm run jest
         # These can't be built in parallel because the cache is a single object and you can't add to it in parallel
         - npm run compile:prod -- --output-path $TRAVIS_BUILD_DIR/travis-build/default
-        - npm run compile:prod -- --env defaultLocale=de --output-path $TRAVIS_BUILD_DIR/travis-build/default-locale
```

Comment:
I have removed the de local run. It wasn't adding much value other than a sanity check that react intl is working.

Reply:
the screenshots for de have also been removed.